web/src/components/layout/demo_layout.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { Switch, Route, withRouter } from 'react-router';
+import { useLocale } from '../locale-helpers';
+import { Spinner } from '../ui/ui';
+
+function DemoLayout() {
+  const [_, toLocaleRoute] = useLocale();
+
+  return (
+    <div>
+      <React.Suspense fallback={<Spinner />}>
+        <Switch>
+          <Route
+            exact
+            path={toLocaleRoute('/demo')}
+            render={() => {
+              return <h1>in get started</h1>;
+            }}
+          />
+          <Route
+            exact
+            path={toLocaleRoute('/demo/language-select')}
+            render={() => {
+              return <h1>in language select</h1>;
+            }}
+          />
+          {/* more routes to be added */}
+          <Route path="*" render={() => <h1>not found</h1>} />
```

Comment:
The last `<Route>` in a `<Switch>` will always be used as the fallback if nothing else matches, so `path="*"` is not needed here. Though I would suggest redirecting this to just the home `/demo` if nothing else is found.